assets/src/dashboard/animations/parts/simpleAnimation.js

```diff
@@ -38,24 +44,37 @@ function SimpleAnimation(animationName, keyframes, timings) {
       if (!target.current) {
         return () => {};
       }
-      const effect = new KeyframeEffect(target.current, keyframes, timings);
+      const effect = createKeyframeEffect(target.current, keyframes, timings);
```

Comment:
This method (`createKeyframeEffect`) is used so we can scrub AMP configurations to their WAAPI equivalent.  The only difference right now is `iterations` wants "infinity" for AMP but wants `Infinity` (the javascript number) for WAAPI.